### PR TITLE
Dropbox provider 

### PR DESCRIPTION
**Pull request by SH2015**
all unnecessary  solution and project files changes removed .

### CODE REVIEW

File: src/AspNet.Security.OAuth.Dropbox/DropboxAuthenticationHelper.cs [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    /// <summary>
+    /// Contains static methods that allow to extract user's information from a <see cref="JObject"/>
+    /// instance retrieved from Dropbox after a successful authentication process.
+    /// </summary>
+    public static class DropboxAuthenticationHelper {
+        /// <summary>
+        /// Gets the identifier corresponding to the authenticated user.
+        /// </summary>
+        public static string GetIdentifier([NotNull] JObject user) => user.Value<string>("id");
+        public static string GetDisplayName ([NotNull] JObject user) => user.Value<string>("display_name");
```
Comment (PinpointTownes) [inline src/AspNet.Security.OAuth.Dropbox/DropboxAuthenticationHelper.cs]: Could you please add a summary for these two methods?

**Comment by PinpointTownes** [comment]
It looks great, I'll merge it ASAP :+1:

**Comment by SH2015** [comment]
Cool , thanks

**Comment by SH2015** [comment]
sure , today I ll add the summary

**Comment by SH2015** [comment]
Summaries added

**Comment by PinpointTownes** [comment]
Rebased and merged: https://github.com/aspnet-contrib/AspNet.Security.OAuth.Providers/commit/cbe593c39063c006ebd25ac1e21e850a24cea103

Thanks for your contribution! :tada:

**Pull request closed without merging**